### ISSUE TITLE
Gateway required?

### ISSUE THREAD

**Issue by rlucian** [AdyRock/com.linktap#22]
Couldn't find some information on the hardware configuration.
Is the linktap HUB required to use the valve with Homey?
Does this app work with G2S, G1S, or both? 
Maybe D1 as well?

best regards and thanks for the app!

**Comment by AdyRock** [comment]
Yes, the gateway is required as LinkTap use a proprietary communications protocol..
The app supports all the LinkTap devices, G1, G1S, G2, G2s and D1 water timers, plus the 1-Zone, 2-Zone and 4-Zone valve controllers, with and without the flow meter.

Hope that helps as they are great products.
